go/test/endtoend/vtcombo/recreate/recreate_test.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2019 The Vitess Authors.
```

Comment:
For new files we should use the current year.

Reply:
Done!